models/articles-model.js

```diff
@@ -4,8 +4,11 @@ const db = require("../db/connection.js");
 //FUNCTION
 function selectArticle(article_id) {
    return db.query("SELECT * FROM articles WHERE article_id = $1", [article_id])
-      .then((article) => {
-        return { article: article.rows[0] };
+     .then(({ rows }) => {
+       if (rows.length === 0) {
+         next(err);
```

Comment:
This next happens in the controller. Notice that we have no 'next' parameter in out model function so there is no way for this to run. What we want to do here is use a `Promise.reject `to trigger an error that will be caught in the controller then passed to `next`